zerothi/sisl
# Sisl visualization module and graphical interface

(This is just a copy of https://github.com/zerothi/sisl/pull/205 using a new branch to allow smooth rebasing)

What is this
-----
This pull request is to get visibility, so that people knows that this is under developement and I can (hopefully) get some feedback about it. Many syntax things still need to be changed, but the concept is already quite complete, so I'd like to get feedback on it to improve weak points.

For those who don't know, this project is an attempt to build a plotting framework and a graphical interface that can really display the great potential of sisl for processing scientific data.

How can I test it
----
It should be straightforward for sisl users to test it, I put everything needed in my branch (although I know that in production I will have to store it elsewhere) so that it is as simple as:

```
git clone -b GUI https://github.com/pfebrer96/sisl.git
cd sisl
pip install -e .[viz]
```

for anyone that would like to test it.

Where do I start
------

The docs are in https://sisl-siesta.xyz/sisl-docs/html/visualization/plotly and they contain several useful (I hope) notebooks that will introduce you to the framework.

**Thanks in advance to any person that takes the time to check it out!**

Cheers!
